test/parallel/test-tls-js-stream.js

```diff
@@ -6,24 +6,19 @@ if (!common.hasCrypto)
 
 const fixtures = require('../common/fixtures');
 
-const assert = require('assert');
 const net = require('net');
 const stream = require('stream');
 const tls = require('tls');
 
-const connected = {
-  client: 0,
-  server: 0
-};
-
 const server = tls.createServer({
   key: fixtures.readKey('agent1-key.pem'),
   cert: fixtures.readKey('agent1-cert.pem')
-}, function(c) {
+}, common.mustCall(function(c) {
   console.log('new client');
-  connected.server++;
+
+  c.resume();
   c.end('ohai');
-}).listen(0, function() {
+})).listen(0, function() {
```

Comment:
I guess you don't need a common.mustCall here because the server has one on connection, and that won't occur if this doesn't run, but adding one might make less reasoning about the test flow necessary.

Reply:
Yes, I didn't add it on purpose for that reason. Will fix tomorrow.